src/com/ms/silverking/cloud/dht/daemon/MessageModule.java

```diff
@@ -585,12 +586,12 @@ public PrimarySecondaryIPListPair getReplicaListPair(long context, DHTKey key, R
             PrimarySecondaryIPListPair  replicaListPair;
             
             if (debug) {
-                System.out.println("\t\tgetReplicaListPair "+ key);
+                Log.fineAsyncf("getReplicaListPair: %s ", key);
             }
             // FUTURE - think about improvements
             replicaListPair = ringMaster.getReplicaListPair(key, ownerQueryOpType);
             if (debug) {
-                System.out.println("\t\t"+ key +"\t"+ replicaListPair +':');
+                Log.fineAsyncf("%s \t %s :", key , replicaListPair);
```

Comment:
debug is off by default. Make Log.warningAsyncf so that the debug flag has an effect